### ISSUE TITLE
Wrong oid/namespace calc

### ISSUE THREAD

**Issue by alangregory** [markabrahams/node-net-snmp#72]
```
{
  ObjectName: 'lldpMIB',
  ModuleName: 'LLDP-MIB',
  MACRO: 'MODULE-IDENTITY',
  'LAST-UPDATED': '',
  ORGANIZATION: '',
  'CONTACT-INFO': '',
  DESCRIPTION: '',
  'OBJECT IDENTIFIER': 'iso std(0) iso8802(8802) ieee802dot1(1) ieee802dot1mibs(1) 2',
  OID: '1.3.6.1.',
  NameSpace: 'iso.org.dod.lldpMIB'
}
```
Looks there is some calculation based that if identifier contains iso, it'll be always the same namespace.

**Comment by alangregory** [comment]
Found the reason, sugestion is to parse like example  `iso std(0) iso8802(8802) ieee802dot1(1) ieee802dot1mibs(1) 2`.
https://github.com/markabrahams/node-net-snmp/blob/1892c2a7bb95cde1b76c0a6d78eae7bcb62459a3/lib/mib.js#L765-L796

**Comment by markabrahams** [comment]
Hi @alangregory - thanks heaps for reporting this - and for the patch.

Have just merged your pull request into master, and published this in v2.5.8 of the npm.
